src/Gyro.cpp

```diff
@@ -27,14 +27,16 @@ void Gyro::startup()
     // Initialize interface to the MPU6050
     LOGV1(DEBUG_INFO, F("GYRO:: Starting"));
     Wire.begin();
+    Wire.setClock(100000);          // Set lowest clock speed to make the communication more reliable
+    Wire.setWireTimeout(3000, true); // Set I2C timeout if the Wire.h calls are hanging due to a HW issue 
```

Comment:
I think you must be using a different library. These functions error out on my MKS build

Reply:
Ok, it pays to read previous conversations. I have Gyro enabled on my MKS, that's probably why it fails? I guess we should switch to the newer library on PIO.

Reply:
@ClutchplateDude if you are trying to use this in the software I2C with MKS, that's not gonna work. You are using a library which is a wrapper around the soft I2C library and acts as an interface for the functions of Wire.h. In this wrapper these new functions are not defined either, because that is based also on the old Wire.h. (See https://github.com/felias-fogg/SlowSoftWire/blob/master/SlowSoftWire.h vs. https://github.com/arduino/ArduinoCore-avr/blob/master/libraries/Wire/src/Wire.h)